unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -133,32 +153,28 @@ void State_Init(BlackBoard bb)
 
     void State_Ground(BlackBoard bb)
     {
-        float dt = Time.deltaTime;
-
-        animation[baseClipsIds.run].normalizedSpeed = bb.movementSpeed / dt * bb.runSpeedFactor;
-        animation[baseClipsIds.walk].normalizedSpeed = bb.movementSpeed / dt * bb.walkSpeedFactor;
-
-        float normalizedSpeed = bb.movementSpeed / dt / MAX_VELOCITY;
+        animation[baseClipsIds.run].normalizedSpeed = bb.movementSpeed / bb.deltaTime * bb.runSpeedFactor;
+        animation[baseClipsIds.walk].normalizedSpeed = bb.movementSpeed / bb.deltaTime * bb.walkSpeedFactor;
```

Comment:
there's no risk of dividing by zero here? XD

Reply:
It shouldn't unless someone force it to be zero :S
Added a check anyway, just in case, better be safe :P